<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteQueueWidget</class>
 <widget class="QWidget" name="RemoteQueueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Queue configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Job &amp;submission command:</string>
        </property>
        <property name="buddy">
         <cstring>edit_submissionCommand</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_submissionCommand">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Request &amp;queue command:</string>
        </property>
        <property name="buddy">
         <cstring>edit_requestQueueCommand</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edit_requestQueueCommand"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&amp;Launch script name:</string>
        </property>
        <property name="buddy">
         <cstring>edit_launchScriptName</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="edit_launchScriptName"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Remote &amp;working directory:</string>
        </property>
        <property name="buddy">
         <cstring>edit_workingDirectoryBase</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="edit_workingDirectoryBase"/>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>SSH connection settings</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Hostname:</string>
           </property>
           <property name="buddy">
            <cstring>edit_hostName</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_hostName"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&amp;User:</string>
           </property>
           <property name="buddy">
            <cstring>edit_userName</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edit_userName"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_identityFile">
           <property name="text">
            <string>&amp;Identity file (private key):</string>
           </property>
           <property name="buddy">
            <cstring>edit_identityFile</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="edit_identityFile"/>
           </item>
           <item>
            <widget class="QToolButton" name="fileButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>&amp;Port:</string>
           </property>
           <property name="buddy">
            <cstring>spin_sshPort</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="keyFileLayout">
           <item>
            <widget class="QSpinBox" name="spin_sshPort"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="push_testConnection">
             <property name="text">
              <string>Test C&amp;onnection...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>SSH &amp;executable:</string>
           </property>
           <property name="buddy">
            <cstring>sshExecutableEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>SCP e&amp;xecutable:</string>
           </property>
           <property name="buddy">
            <cstring>scpExecutableEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="sshExecutableEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="scpExecutableEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="push_sleepTest">
          <property name="text">
           <string>S&amp;ubmit test job...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Job &amp;cancel command:</string>
        </property>
        <property name="buddy">
         <cstring>edit_killCommand</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_killCommand"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Default &amp;walltime limit:</string>
        </property>
        <property name="buddy">
         <cstring>wallTimeHours</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="wallTimeHours">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> h</string>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="wallTimeMinutes">
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Queue update interval:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="updateIntervalSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Launcher template</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QPushButton" name="templateHelpButton">
        <property name="text">
         <string>Template &amp;help</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="text_launchTemplate">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_submissionCommand</tabstop>
  <tabstop>edit_killCommand</tabstop>
  <tabstop>edit_requestQueueCommand</tabstop>
  <tabstop>updateIntervalSpin</tabstop>
  <tabstop>edit_launchScriptName</tabstop>
  <tabstop>edit_workingDirectoryBase</tabstop>
  <tabstop>wallTimeHours</tabstop>
  <tabstop>wallTimeMinutes</tabstop>
  <tabstop>sshExecutableEdit</tabstop>
  <tabstop>scpExecutableEdit</tabstop>
  <tabstop>edit_hostName</tabstop>
  <tabstop>edit_userName</tabstop>
  <tabstop>spin_sshPort</tabstop>
  <tabstop>push_testConnection</tabstop>
  <tabstop>push_sleepTest</tabstop>
  <tabstop>text_launchTemplate</tabstop>
  <tabstop>templateHelpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
